### PR TITLE
Updated sessions and cache configs

### PR DESCRIPTION
**Pull request by waseem-omar**

### CODE REVIEW

**Comment by waseem-omar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;PMAFe7dG&#x2F;1033-1-53-resolve-redis-timeout-error-cycle-25-cycle-26"> (1) 53 - Resolve redis timeout error [Cycle 25] [Cycle 26]</a>

**Comment by GhalebKhaled** [comment]
@waseem-omar I can see in `registration/views.py` in `dispatch` method line 528 on this branch, the current selected cache is `file_storage` maybe we need to replace it with redis cache (the default one), Thoughts?

**Comment by GhalebKhaled** [comment]
@waseem-omar Just ignore my comment, `file_storage` is not defined so it will go to `default` cache

File: settings/base.py [hunk 1]
```diff
@@ -286,37 +286,31 @@ def is_environ_true(name):
...
         'TIMEOUT': 60 * 60 * 24,  # 1 day
     },
     'staticfiles': {
-        'BACKEND': 'redis_cache.RedisCache',
-        'LOCATION': ':'.join([unicode(REDIS.hostname), unicode(REDIS.port)]),
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': REDIS_URL,
         'OPTIONS': {
+            # "SOCKET_CONNECT_TIMEOUT": 5,  # in seconds
             'DB': REDIS_DB,
```
Comment (danaspiegel) [inline settings/base.py]: @waseem-omar can we set this to a different DB (defined as a static definition above)? We can use DB 1 for static files.

**Pull request merged**